### PR TITLE
Fix gRPC docs (wrong command used, removed duplication from java README)

### PR DESCRIPTION
**Pull request by ignasi35**

### CODE REVIEW

File: grpc-example/grpc-example-java/README.md [hunk 1]
```diff
@@ -1,107 +1,22 @@
 # Lagom gRPC Example (Java)
 
-This project demonstrates the usage of [akka-grpc](https://github.com/akka/akka-grpc) into Lagom.
+## Guide
```
Comment (ignasi35) [inline grpc-example/grpc-example-java/README.md:3]: This gRPC guide was originally a `README` and was then migrated to paradox to publish on techhub. When migrating, this README was left over. In the scala example, though, the README was cleaned.

File: grpc-example/grpc-example-java/docs/src/main/paradox/index.md [hunk 2]
```diff
@@ -117,7 +117,7 @@ You can test the gRPC endpoint using [grpcc](https://github.com/njpatel/grpcc).
 certificates, you will have to export and trust the CA certificate:
 
 ```bash
-keytool -export -alias playgeneratedCAtrusted -keystore target/dev-mode/generated.keystore  -storepass "" -file trustedCA.crt
+keytool -export -alias sslconfig-selfsigned  -keystore target/dev-mode/selfsigned.keystore  -storepass "" -file trustedCA.crt
```
Comment (ignasi35) [inline grpc-example/grpc-example-java/docs/src/main/paradox/index.md:120]: This is the actual fix.

File: grpc-example/grpc-example-scala/docs/src/main/paradox/index.md [hunk 3]
```diff
@@ -119,7 +119,7 @@ You can test the gRPC endpoint using [grpcc](https://github.com/njpatel/grpcc).
 certificates, you will have to export and trust the CA certificate:
 
 ```bash
-keytool -export -alias playgeneratedCAtrusted -keystore target/dev-mode/generated.keystore  -storepass "" -file trustedCA.crt
+keytool -export -alias sslconfig-selfsigned  -keystore target/dev-mode/selfsigned.keystore  -storepass "" -file trustedCA.crt
```
Comment (ignasi35) [inline grpc-example/grpc-example-scala/docs/src/main/paradox/index.md:122]: This is the actual fix.

**Review by ignasi35 — COMMENTED** [review]
left notes to ease review.

**Review by marcospereira — APPROVED** [review]

**Comment by ignasi35** [comment]
Backported `1.5.x` 0c7a861b58379cab47bf4ebaf913985444e74157

**Pull request merged**